lib/resty/healthcheck.lua

```diff
@@ -30,6 +30,7 @@ local DEBUG = ngx.DEBUG
 local ngx_log = ngx.log
 local tostring = tostring
 local ipairs = ipairs
+require("table.nkeys")
```

Comment:
Don't understand the use of this line of code

Reply:
The `table.nkeys` needs to be required before use. Previously in APISIX, we already require this method. But when running in the CI, we need to require it in this file.